x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -49,6 +49,9 @@
     public static final String DATAFEED_FREQUENCY_MUST_BE_MULTIPLE_OF_AGGREGATIONS_INTERVAL =
             "Datafeed frequency [{0}] must be a multiple of the aggregation interval [{1}]";
     public static final String DATAFEED_ID_ALREADY_TAKEN = "A datafeed with id [{0}] already exists";
+    public static final String DATAFEED_NEEDS_REMOTE_CLUSTER_SEARCH = "Datafeed [{0}] is configured with a remote index pattern(s) {0}" +
```

Comment:
Perhaps include the node Id in the message so the user knows which node to update?

Reply:
I think we should still keep `Please enable cluster.remote.connect for all machine learning nodes` though.

Maybe something like `Please enable cluster.remote.connect for all machine learning nodes (currently disabled on [<nodename>])`.